var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -32,14 +32,16 @@ class Libfabric(AutotoolsPackage):
     homepage = "https://libfabric.org/"
     url      = "https://github.com/ofiwg/libfabric/releases/download/v1.5.0/libfabric-1.5.0.tar.gz"
 
+    version('1.5.3', '1fe07e972fe487c6a3e44c0fb68b49a2')
     version('1.5.0', 'fda3e9b31ebe184f5157288d059672d6')
+    version('1.4.2', '2009c8e0817060fb99606ddbf6c5ccf8')
 
     fabrics = ('psm',
                'psm2',
                'sockets',
                'verbs',
                'usnic',
-               'mxm',
+               'mlx',
```

Comment:
MLX is how libfabrics calls UCX. So for sure the dependency has to be updated.

Reply:
Thanks - I added the ucx dependency when mlx is enabled.